book/src/backends/estark.md

```diff
@@ -0,0 +1,3 @@
+# eSTARK
+
+Integrated support for [eSTARK](https://eprint.iacr.org/2023/474) with the goldilocks field is under development.
```

Comment:
```suggestion
Integrated support for [eSTARK](https://eprint.iacr.org/2023/474) with the Goldilocks field is under development.
```

Reply:
Maybe add references to eSTARK and Goldilocks?

Reply:
eSTARK is there, I couldn't find a good source for goldilocks. @georgwiese any thoughts?

Reply:
That could work!
>Yes, I *still* haven't published anything about the Goldilocks field. 😅 For a citation, you could link to the original issue and make the author "The Polygon Zero Team" or something like that. Will that work for you?
https://github.com/mir-protocol/plonky2/issues/1